### PR TITLE
fix(ui): Increase hit area for sidebar buttons

### PR DESCRIPTION
**Pull request by juliendonck**
This increases the clickable area of sidebar items to the size where the indicator shows up.  (closes #664 and #1296)
sidebar-hover

### CODE REVIEW

File: ui/DesignSystem/Component/Sidebar.svelte [hunk 1]
```diff
@@ -111,9 +111,11 @@
     <li
```
Comment (juliendonck) [inline ui/DesignSystem/Component/Sidebar.svelte:111]: @cloudhead I could wrap the tooltip around the `<li>` and change the `ul` and `li` to divs so the markup is correct. But it's complicated cause some of the markup is replicated in the `ConnectionStatusIndicator` component.

**Comment by juliendonck** [comment]
The last commit move the tooltip to outside the sidebar. I can live with this, don't wanna overcomplicate the tooltip logic for this one use case.

File: ui/DesignSystem/Component/ConnectionStatusIndicator.svelte [hunk 2]
```diff
@@ -18,45 +18,60 @@
...
   }
 </style>
 
 <Remote {store} let:data={status}>
   <div>
     {#if status.type === StatusType.Online}
       <Tooltip value={`You’re connected to ${peerCount(status.connected)}`}>
-        <!-- svelte-ignore a11y-missing-attribute -->
-        <a>
+        <div class="item indicator" data-cy="search">
```
Comment (MeBrei) [inline ui/DesignSystem/Component/ConnectionStatusIndicator.svelte:53]:
```suggestion
        <div class="item indicator" data-cy="connection-status">
```

also for the changes below

**Review by MeBrei — APPROVED** [review]
Looking good!

**Review by xla — APPROVED** [review]
💶 👝 👜 🕸

**Review by rudolfs — APPROVED** [review]

**Pull request merged**